README.md

```diff
@@ -11,29 +11,77 @@ The Readme is still a work in progress. It actually won't get you very far, beca
 
 ### Setup
 1. Clone the repo.
-
     ```shell
     git clone https://github.com/silcam/dulu.git
+    ```
 
-2. Create databases dulu_dev and dulu_test. See database.yml for the username and password to use.
+1. Create databases dulu_dev and dulu_test. See database.yml for the username and password to use.
+   
+    ```shell
+    rails db:create
+    ```
+   
+   Note: if you also need to drop the db, you can first run
 
-3. Install bundler if you don't already have it
+    ```shell
+    rails db:drop (if necessary)
+    ```
+
+1. Install bundler if you don't already have it
 
     ```shell
     gem install bundler
+    ```
 
-4. Install the necessary gems
+1. Install the necessary gems
 
     ```shell
-    cd dulu
     bundle install
+    ```
+
+1. yarn install
 
-5. Initialize the development database
+    ```shell
+    yarn install
+    ````
+
+   If you don't have it, install [yarn](https://classic.yarnpkg.com/) first.
+
+1. Initialize the development database by loading the schema
 
     ```shell
     rails db:schema:load
+    ```
+
+### Running the Tests
+
+If you just want to run the tests, you can after a few more steps. The test database does not need to be seeded.
 
-6. Start the server. Access Dulu at http://localhost:3000
+1. In `secrets.yml`, add Rails.application.secrets.gmail_username, set to something fake.
 
+1. Run tests.
+
+   See the definitions in `package.json` about what to do, but for example you can run `yarn test:most` to
+    test most things. (You may also need to install [cypress](https://www.cypress.io/) to do that.)
```

Comment:
`test:most` is all the tests except for cypress. I'm also pretty sure there's no need to install cypress separately from `yarn install` since it's in the package.json.

Reply:
I'll change it. I already had cypress installed so I wasn't sure if you needed to do any manual steps.